polaris/polaris/sep24/deposit.py

```diff
@@ -313,6 +313,9 @@ def deposit(account: str, request: Request) -> Response:
     stellar_account = request.POST.get("account")
     lang = request.POST.get("lang")
     sep9_fields = extract_sep9_fields(request.POST)
+    claimable_balance_supported = bool(
+        request.POST.get("claimable_balance_supported") == "true"
```

Comment:
Lets make sure our processing of the request parameter is in-line with the changes outlined in the SEP. Does the request value _need_ to be `true`? Can it be `1`, `True`, or `TRUE`? Gotta be careful with these string comparisons.

Reply:
When the wallet sends boolean `true` in the POST req body from its end Python reads it as string `"true"`.
So I wanted to normalize it to a boolean on Polaris' end.

I can make it a bit more forgiving by checking if its returned as `1`, `True`, or `TRUE` (the latter two via python's [lower](https://www.kite.com/python/docs/builtins.str.lower))

Reply:
I believe JSON has native `true/false` values, so I would think Django would parse those as native Python booleans.

But that was kinda beside my point. My point was that the parsing logic needs to match the SEP. So if the SEP specifically mentions `claimable_balance_supported` _must_ have a value of `"true"`, then what you have is good 👍

Reply:
Ok, I took a look at the SEP PR for CBs and it looks like this is how the parameter is defined:
<img width="977" alt="Screen Shot 2020-12-01 at 12 56 09 PM" src="https://user-images.githubusercontent.com/10968980/100796478-907b9180-33d5-11eb-9ae8-e670c0ae4fbc.png">

I think you need to do some digging into what a boolean looks like in a POST request. Does it come in as a string, `"true"`, like you're checking for now? Or does it come in as a native python boolean, `True`?